framework/src/modules/interoperability/mainchain/module.ts

```diff
@@ -79,11 +86,58 @@ export class MainchainInteroperabilityModule extends BaseInteroperabilityModule
 		this.interoperableCCCommands,
 		this.internalMethod,
 	);
+	private readonly _crossChainUpdateCommand = new MainchainCCUpdateCommand(
+		this.stores,
+		this.events,
+		this.interoperableCCMethods,
+		this.interoperableCCCommands,
+		this.internalMethod,
+	);
+	private readonly _messageRecoveryCommand = new MainchainMessageRecoveryCommand(
+		this.stores,
+		this.events,
+		this.interoperableCCMethods,
+		this.interoperableCCCommands,
+		this.internalMethod,
+	);
+	private readonly _stateRecoveryCommand = new StateRecoveryCommand(
+		this.stores,
+		this.events,
+		this.interoperableCCMethods,
+		this.interoperableCCCommands,
+		this.internalMethod,
+	);
+	private readonly _terminateSidechainForLivenessCommand = new TerminateSidechainForLivenessCommand(
+		this.stores,
+		this.events,
+		this.interoperableCCMethods,
+		this.interoperableCCCommands,
+		this.internalMethod,
+	);
 
 	// eslint-disable-next-line @typescript-eslint/member-ordering
 	public commands = [
-		this._sidechainRegistrationCommand,
+		this._crossChainUpdateCommand,
 		this._messageRecoveryInitializationCommand,
+		this._messageRecoveryCommand,
+		this._sidechainRegistrationCommand,
+		this._stateRecoveryCommand,
+		this._terminateSidechainForLivenessCommand,
+	];
+	// eslint-disable-next-line @typescript-eslint/member-ordering
+	public crossChainCommand = [
```

Comment:
Would it be better if called `crossChainCommands` if it's an array?

Reply:
yes, ill create another issue. This was defined in base class